api/ClipboardItem.json

```diff
@@ -210,11 +210,9 @@
             "spec_url": "https://w3c.github.io/clipboard-apis/#optional-data-types",
             "support": {
               "chrome": {
-                "version_added": "121"
-              },
-              "chrome_android": {
-                "version_added": false
+                "version_added": "124"
               },
+              "chrome_android": "mirror",
```

Comment:
Our data says the API isn't supported in Chrome Android.

Reply:
Oh really. I trust your data more than my knowledge on this, but let's ask @tayloregivens in case she can provide insights from the implementation side of things.

Reply:
Oh really. I trust your data more than my knowledge on this, but let's ask @tayloregivens in case she can provide insights from the implementation side of things.

Reply:
Haven't heard back from Taylore yet, but the chromestatus.com entry does confirm this is available on Android: https://chromestatus.com/feature/5125790490427392.
Perhaps your data only applied to the supports_static method?

Reply:
@snianu would appreciate if you could provide some clarity on this. I'm not familiar with whether the supports() static method works on Chrome Android.

Reply:
@snianu thanks for your guidance. Dan walked me though setting up Android VM, making
[Glitch demo page](https://abalone-basalt-raver.glitch.me/)
, and going through Chromium idl file to review changes to the ClipboardItem and supports() method. We tried test page with Android VM and Chrome Canary, and
supports() is supported on Chrome for Android
likely in the same version it was supported on desktop, 121.

Reply:
Thanks @tayloregivens for testing. @queengooborg could this be a bug on the BCD Collector side? The `supports()` static method appears to be working on Chrome Android.